waku/waku_api/rest/doc/relayapi_subscriptions.yaml

```diff
@@ -0,0 +1,50 @@
+# /relay/v1/subscriptions:
+post: # post_waku_v2_relay_v1_subscriptions
+  summary: Subscribe a node to an array of topics
+  description: Subscribe a node to an array of PubSub topics.
+  operationId: postSubscriptions
+  tags:
+    - relay
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type array:
+          items:
+            $ref: './schemas/apitypes.yaml#/PubsubTopic'
+  responses:
```

Comment:
How is partial error handled? 
Let's say if we request to subscribe to 2 topics and one of them fails and other succeeds, how is the response returned to caller? Better to include this information as well.

Similar case needs to be handled in all API's where multiple operations are executed on server side.